Changelog.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 20231009
+
+#### New schemas:
+- LoyaltiesListMemberRewardsRequestQuery
+- LoyaltiesGetPointsExpirationRequestQuery
+- LoyaltiesGetPointsExpirationResponseBody
+- LoyaltiesListCardTransactionsRequestQuery
+- LoyaltiesListCardTransactionsResponseBody
+- LoyaltyCardTransactionsType
+- SimpleLoyaltyVoucher
+- LoyaltyCardTransaction
+- LoyaltyCardTransactionsFields
+- LoyaltiesExportCardTransactionsRequestBody
+- LoyaltiesExportCardTransactionsResponseBody
+- RewardAssignment
+- Reward
+- RewardTypeCoin
+- RewardTypeMaterial
+- RewardTypeCampaign
+- RewardType
+- LoyaltiesAddOrRemoveCardBalanceResponseBody
+- LoyaltiesTransferPointsResponseBody
+- LoyaltiesTransferPoints
+- LoyaltiesTransferPointsRequestBody
+- LoyaltiesListMemberRewardsResponseBody
+- LoyaltiesAddOrRemoveCardBalanceRequestBody
+- PointsExpirationTypes
+
+#### Schemas changes
+- /v1/loyalties/{campaignId}/members/{memberId}/balance
+    - new request schema `LoyaltiesAddOrRemoveCardBalanceRequestBody` (old `8_req_add_remove_points_balance`)
+    - new response schema `LoyaltiesAddOrRemoveCardBalanceResponseBody` (old `8_res_add_remove_points_balance`)
+- /v1/loyalties/members/{memberId}/balance
+    - new request schema `LoyaltiesAddOrRemoveCardBalanceRequestBody` (old `8_req_add_remove_points_balance`)
+    - new response schema `LoyaltiesAddOrRemoveCardBalanceResponseBody` (old `8_res_add_remove_points_balance`)
+- /v1/loyalties/{campaignId}/members/{memberId}/transfers
```

Comment:
Missing body params attributes descriptions

![image](https://github.com/voucherifyio/voucherify-openapi/assets/35666945/ef767116-8cf8-42f8-9a43-4c21389892a7)

Reply:
and response body attributes description: 
![image](https://github.com/voucherifyio/voucherify-openapi/assets/35666945/9e0f38b8-62ce-4532-b425-72374b7cfdd0)